src/main/java/com/epam/prejap/teatrees/game/Printer.java

```diff
@@ -29,4 +39,35 @@ void endRow() {
     void border(int width) {
         out.println("+" + "-".repeat(width) + "+");
     }
+
+    /**
+     * Prints the concluding message about achieved score. In case of new record prints additional message.
+     *
+     * @param isNewBestRecord indicates whether new score is the best yet for the Player
+     * @param score           user's score, got during the game
+     * @param nameOfThePlayer
+     */
+    public void printFinalMessage(List<Record> scores, boolean isNewBestRecord, int score, String nameOfThePlayer) {
```

Comment:
That's a lot of parameters. Why not encapsulate them in a new class that would have custom toString containing all logic from this method?

Reply:
That could be a good idea, but there are only 4 parameters (just with long title), and actually that's the only place these parameters are needed.